### PR TITLE
Clean up for new domain name changes.

### PR DESCRIPTION
**Pull request by pwei1018**
*Issue #:* [/bcgov/entity###](https://github.com/bcgov/entity/issues/15877)

*Description of changes:*
1. Clean up unused files;
2. Enable cloud build PR review feature;
3. Move firebase and cloud build configuration to UI CD flow in bcregisty-sre repo;
4. Update environment variables' value;

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: --pr-595-39f7f361.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://namerequest-dev--pr-595-39f7f361.web.app

**Comment by pwei1018** [comment]
Temporary Url for review: https://namerequest-dev--pr-595-39f7f361.web.app

**Review by ozamani9gh — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-request",
-  "version": "4.0.0",
+  "version": "4.0.1",
```
Comment (severinbeauvais) [inline package.json:3]: This is a significant change -- I recommend updating the minor release (or even major release) number.

File: devops/vaults.json [hunk 2]
```diff
@@ -1,55 +0,0 @@
-[
```
Comment (severinbeauvais) [inline devops/vaults.json:1]: Don't we still need this file to build the env variables from 1Password?

Comment (pwei1018) [reply devops/vaults.json:1]: no, don't need it in UI projects.

**Review by bolyachevets — APPROVED** [review]

File: devops/vaults.json (hunk 2 above)
Comment (severinbeauvais) [reply devops/vaults.json:1]: OK. But you'll have to explain to me (or team) how it works now :)

File: package.json (hunk 1 above)
Comment (severinbeauvais) [reply package.json]: Thanks.

**Review by severinbeauvais — APPROVED** [review]

**Comment by pwei1018** [comment]
Temporary Url for review: https://namerequest-dev--pr-595-39f7f361.web.app

**Pull request merged**